sensor/common/processsignal/pipeline_test.go

```diff
@@ -6,17 +6,183 @@ import (
 	"time"
 
 	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/process/filter"
 	"github.com/stackrox/rox/pkg/sync"
 	"github.com/stackrox/rox/sensor/common"
 	"github.com/stackrox/rox/sensor/common/clusterentities"
 	"github.com/stackrox/rox/sensor/common/detector/mocks"
 	"github.com/stackrox/rox/sensor/common/message"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"go.uber.org/mock/gomock"
 )
 
-func TestProcessPipelineOffline(t *testing.T) {
+const (
+	containerID1      = "1e43ac4f61f9"
+	containerID2      = "2e43ac4f61f9"
+	containerID3      = "3e43ac4f61f9"
+	deploymentID1     = "mock-deployment-1"
+	deploymentID2     = "mock-deployment-2"
+	deploymentID3     = "mock-deployment-3"
+	outputChannelSize = 2
+)
+
+func TestProcessPipelineOfflineV3(t *testing.T) {
```

Comment:
Should online and offline events make any difference in this tests? If the notify is skipped when the FF is enabled, the tests should still pass without the `online` and `offline` event functions, no?

Reply:
They don't make any difference 🙂 I just added them to illustrate how the Pipeline would be called in a real scenario. I've added a comment in the tests explaining this.